internal/conf/conf.go

```diff
@@ -61,6 +61,7 @@ type EnvironmentConfig interface {
 	Config
 
 	Prefix() string
+	Strip() string
```

Comment:
I also had this thought, I almost named this function "ToStrip()" (not a great name) because the actual prefix stripping does not occur in the context of Strip(). Instead, I opted for a consistent function name ("prefix" : "Prefix()" :: "strip" : "Strip()").

I don't mind changing 'strip' to 'cutset', but I will wait to see what Rohan thinks, as he suggested "strip".

Reply:
Separating my opinion into two pieces...

### Key used in config file

- I had trouble coming up with a proposal here in the first place and I'd be fine with changing 'strip' to something else.
- I'm not a fan of 'cut' there as it seems more obscure to me than 'strip'.  I'd expect that 'trim' might be the leading alternative.
- The terms used in the Go standard library shouldn't influence us much here.  The config file is for users/administrators of the tool, those are not supposed to require any knowledge of the programming language used to develop the tool.

### Method name in interface

If there were no connection with the config file, it would be right to avoid a name of `Strip`.  However, so far I think these names in the interface exactly match up with the config file.  It would be a shame to start deviating from that, as whenever you're working with the interface you'll have to remember the difference between how it's named in the config and how it's named on the interface.

In cases like these, where some object maps to an external piece of structured data, I think it's generally accepted that you'll pick names aligned with the underlying data even if that's not the way you would name things under other circumstances when working in a certain language.